package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "5.3.0",
+  "version": "5.2.3",
```

Comment:
Why's the SDK version rolled back for this PR?

Reply:
Good spot.